src/main/java/seedu/address/ui/GroupListPanel.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+//import org.fxmisc.easybind.EasyBind;
```

Comment:
please don't leave around commented unused code. delete them and commit. revert later to the commit if you want to restore